### ISSUE TITLE
Strange text editing behavior of note elements

### ISSUE THREAD

**Issue by terfloth** [Yakindu/statecharts#1505]
The editing behavior of note elements is annoying (reported for MacOS). The text editing area should fill the complete space of the note element (except for a border).

**Comment by BeckmaR** [comment]
Same on Linux. They are nearly impossible to use.

**Comment by realHz** [comment]
[notes woes.zip](https://github.com/Yakindu/statecharts/files/1214638/notes.woes.zip)

**Comment by SimonEit** [comment]
Yes, same goes for text fields / text notes. I tried using them extensively while creating a tutorial series. They are neither resizable nor does the line break work correctly. Even getting the cursor set to the place you want is really hard.
